Auth/Auth.Application/Features/User/Commands/LoginUser/LoginUserCommandHandler.cs

```diff
@@ -13,20 +12,17 @@ namespace Auth.Application.Features.User.Commands.LoginUser
     public class LoginUserCommandHandler : IRequestHandler<LoginUserCommand, string>
     {
         private readonly IUserRepository userRepository;
-        private readonly IAccountDataRepository accountDataRepository;
         private readonly ITokenService tokenService;
         private readonly IMapper mapper;
         private readonly IValidator<LoginUserCommand> validator;
```

Comment:
mostly on my projects the naming for private fields was `_mapper` so we don't need `this.`